### DISCUSSION TITLE
Issue to upload my IPTV list channels on Fermata Auto

### DISCUSSION THREAD

**Discussion by Montassirovsky** [AndreyPavlenko/Fermata#202] [General]
Hi. 
I have an issue to upload my iptv list channels in Fermata Auto. I have already uploaded a free IPTV list channels and it works well. However my paid iptv list channels which is including beinsport channels is not working.
Anyone can help me how is the process to upload an iptv list channels on Fermata Auto?

**Comment by AndreyPavlenko** [comment]
Hi,

What exactly does not work? You are not getting the list of channels or it does not play?

**Reply by Aydeh53** [reply]
Hello Andrey.  First of all, I would like you to know that I am grateful for your efforts and I wish you continued success.  I am experiencing the same problem that your friend is experiencing.  The URL address I obtained from my paid IP TV provider does not work in fermata auto.  I don't know if it could be because paid IP TVs are encrypted, but even if there is such a thing, I couldn't find a place to enter the password on Fermata.  If you have any information or work on this subject, I would appreciate it if you could share it with us.  Thank you in advance and wish you good work.

**Reply by CMAC1986** [reply]
This is exactly my issue. My provider does not provide an m3u file. They tell me to just enter the URL into the browser and it should download but it does not. I was able to download the m3u around a year ago. Luckily I was able to edit that in txt form and some of the channels work while others are invalid due to the URLs being updated. Adding my URL into the app does not load. The older m3u file that I did download a year ago was around 150mb!!. This could be the issue as to why they won't load into the app. Hope this helps

**Reply by Aydeh53** [reply]
I added the URL address I received from my provider to the browser on my phone, said OK and it started downloading, at this point it gave an error and I continued with the save option and it downloaded to my phone as an m3u file.  I tried it today and now I can use it without any problems.

**Comment by Montassirovsky** [comment]
Hello Andrey

Thanks for your reply

Yes Exactly. I have actually my own IPTV however it's not supported by
Fermata App.

Could you please help with this?

Thanks

**Reply by AndreyPavlenko** [reply]
Could you please describe what does not work? Are you getting an empty list?

**Comment by Montassirovsky** [comment]
Hello Andrey

After doing copy past to the link of my IPTV list channels on Fermata Auto
App and giving a name in the title, a new folder created with the name that
I have given in the title. However,  this folder appears empty without any
channel.

Please find attached what I see after doing copy past to the link of my
IPTV list channels on Fermata Auto App.

Awaiting your reply with a solution.

Regards

**Reply by AndreyPavlenko** [reply]
Have you tried to download the playlist to a file and specify this file in the app, instead of the link?

**Comment by Montassirovsky** [comment]
Hello Andrey

No honestly  I don't know how to download my iptv link. I know only doing
copy past on Fermata Auto App

Can you help me with that?

Also can you text me on WhatsApp rather than email?

There is my WhatsApp phone number:  +212 663638973

Awaiting your reply

Regards

**Reply by AndreyPavlenko** [reply]
Just click on the link or open it in a browser on your phone.

**Reply by TtheSshadow** [reply]
Were you ever able to resolve your issue? I am experiencing the same problem. The free IPTV urls work and populate Groups with channels. When I try to load the url from my IPTV provider I receive a timeout error.

**Reply by AndreyPavlenko** [reply]
Perhaps, the playlist is too large and/or the download speed is to slow. That's why you are getting timeout. As a workaround, you can download the playlist into a file and add this file as a TV source.

**Comment by TtheSshadow** [comment]
This is what I receive when attempting to use load the url...

**Comment by TtheSshadow** [comment]
Whenever I load the url provided by my IPTV provider, it stops downloading
and restarts the entire process over again.

**Reply by AndreyPavlenko** [reply]
How do you download?

**Reply by TtheSshadow** [reply]
I copy the url link my IPTV provider gave me and open it in a Chrome browser. After a few seconds a download starts.

**Reply by AndreyPavlenko** [reply]
If you can't download the playlist even with a web browser, then, probably, the problem is no the provider side.

**Comment by TtheSshadow** [comment]
After talking to their support team they sent me a different link that only
had the live streaming TV channels without the video on demand so that made
the playlist a lot smaller to load and so now it works with just the
streaming TV and they remove the video on demand

**Comment by onemsiz** [comment]
@AndreyPavlenko

Is it possible for you to add Xtream code support to IPTV? Nowadays, most IPTV providers do not provide M3U files and links anymore. As you can see from the img at below, you can watch streams using Xtream codes.  If you can integrate this into an IPTV application, that would be great. 
![iptv](https://github.com/AndreyPavlenko/Fermata/assets/119848360/1a24bda3-2154-4c80-b262-ce9feae67a80)

**Reply by AndreyPavlenko** [reply]
Is there any documentation for this?

**Reply by sinancelik1987** [reply]
Xtream IPTV is a bit different from normally free IPTV. Xtream needs also username and password. Can you upgrade Fermata Auto with Xtream IPTV feature. Thank you.

**Comment by CMAC1986** [comment]
Sorry to hijack an old post. I am also having issues similar to the OP. Basically my provider does not provide an m3u file so I can't add it to fermata. I've asked them and they won't provide it. I've tried downloading it via the URL but it doesn't do anything. The issue for me is I can't type the URL into fermata because the keyboard doesn't appear. Any help would be appreciated. Thanks

**Reply by AndreyPavlenko** [reply]
The keyboard should appear when you click on the text field. If you are trying to do that from the car screen, just use your phone.

**Reply by CMAC1986** [reply]
Yeah I'm trying to do it from the car screen. Not to worry, without the m3u I can't watch anyways as for some reason entering the URL it won't load regardless. Thanks again

**Reply by chevyman23** [reply]
Any idea how to get the keyboard to pop up when it doesn't? When I tap in the location field it doesn't show up on the car screen or the phone. I have all my lists on my phone, but they won't transfer to the car either.

**Reply by AndreyPavlenko** [reply]
On the car screen you see absolutely the same app and absolutely the same playlists. Just add the playlists on your phone.

**Reply by chevyman23** [reply]
@AndreyPavlenko Thabk you. I figured out I had 2 different versions on my phone and was using the wrong one. Any good, free lists you can share?

**Comment by Frederic60127** [comment]
Bonjour j'ai un problème avec fermata auto, lorsque je mets le lien m3u et mon username dans fermata auto je n'ai aucune liste de chaîne qui apparaît. Avez vous une solution ? 
![Screenshot_20260131_111248](https://github.com/user-attachments/assets/d652a3c0-f716-4271-9f24-76fbcdf090e8)

**Comment by Ojaidi000** [comment]
https://xtreamcode.ex/player_api.php?username=Mike&password=1234&action=get_vod_categories
